github/plugin.py

```diff
@@ -82,8 +82,23 @@ def __init__(self):
                 "user_data_required": True,
                 "help": "Delete a GitHub repository.",
                 "enabled": True     # It is HIGHLY recommended you have auth enabled for this!!
+            },
+            "!InviteUser": {
```

Comment:
Actually -- now that I'm thinking about this -- we should remove this -- or rather, make it a shortcut to `!AddUserToTeam`.

I feel that it is good practice to have all members exist within a team. Perhaps this can be modified where you can configure a `default_team` to add new members to.